mlflow/server/js/craco.config.js

```diff
@@ -194,7 +190,19 @@ module.exports = function ({ env }) {
         ],
       ],
     },
-    ...(proxyTarget && {
+    ...(process.env.MLFLOW_DEV_PROXY_MODE && {
```

Comment:
This code block is not required here but shouldn't do any harm. We will OSS-guard it on next export.

Reply:
Makes sesnse, thanks for the reivew!